.github/workflows/openapi-docs-pr.yml

```
@@ -0,0 +1,18 @@
+name: Test OpenAPI doc is valid on Pull Requests
+
+on:
+  pull_request:
+    paths:
+      - "schemas/v1alpha/openapi.yml"
```

Make this a wildcard so it picks up others in the future

```suggestion
      - "schemas/*/openapi.yml"
```